test/libsolidity/syntaxTests/dataLocations/externalFunction/external_function_return_parameters_no_data_location.sol

```diff
@@ -2,4 +2,4 @@ contract C {
     function i() external pure returns(uint[]) {}
 }
 // ----
-// TypeError: (52-58): Storage location must be: "memory" for parameter in external function, but none was given.
+// TypeError: (52-58): Storage location must be one of "memory" for parameter in external function, but none was given.
```

Comment:
Actually it seems we should also have a special case for only a single element... (`must be "memory"`)

Reply:
Or perhaps it is better to just concatenate with `" or "` instead of `", "`? In case there are three options, we have two `or`s, but I think that's fine.

Reply:
I agree, that sounds best.